content/working_with/console/widgets/deploymentsViewDrilledDown.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: bt_wiki
+title: Deployments View (drilled-down)
+category: Widgets
+draft: false
+---
+
+The _Deployments View (drilled-down)_ widget is a drilled-down version of
+[the regular Deployments View widget]({{< relref "working_with/console/widgets/deploymentsView.md" >}}).
+
+It should be used only inside pages that were drilled-down to when using the
+Deployments View widget. Using this widget on other pages will result in errors
+begin shown.
```

Comment:
I think you meant "errors being shown"?
seems like some typo.

Reply:
Yes, definitely a typo :slightly_smiling_face: I'll correct it

Reply:
Done :slightly_smiling_face: